demisto_sdk/tests/integration_tests/secrets_integration_test.py

```diff
@@ -67,3 +68,39 @@ def test_integration_secrets_negative(mocker):
     assert "Dynamics365ForMarketingEmail" in result.stdout
     assert "Remove or whitelist secrets in order to proceed, then re-commit" in result.stdout
     assert result.stderr == ""
+
+
+def test_integration_secrets_positive_s(mocker):
```

Comment:
thats not an integration test, it is a legit test but should go in the secrets unit tests